monkestation/code/modules/jobs/job_types/mailman.dm

```diff
@@ -1,16 +1,15 @@
-/datum/job/gimmick/mailman
+/datum/job/mailman
 	title = "Mailman"
 	flag = MAILMAN
 	department_head = list("Head of Personnel")
 	department_flag = CIVILIAN
 	faction = "Station"
-	outfit = /datum/outfit/job/gimmick/mailman
+	outfit = /datum/outfit/job/mailman
```

Comment:
You will need to update the outfit datum below as well if you are removing gimmick from its path.

Reply:
okay, where do i find the "/datum" part?

Reply:
okay i think i did it, check if i did it right

Reply:
okay i know how to do it i just dont know how to add something to the PR

Reply:
Additional needs on this file:

- Setting the job display_color, check cargo tech or shaft miner for color

- you will need to set the display_order variable, which will also need to be defined.  check cargo tech or shaft miner for this variable, and update the job defines file accordingly.

- rename the file to mailman.dm

Reply:
does that look good?

Reply:
this file is fine now.  Please make the changes to code\__DEFINES\jobs.dm and monkestation\code\modules\jobs\mail_goodies\job_mail.dm

Reply:
weird i thought i had fixed that

Reply:
okay im having serious problems on how to place changes on the same PR and branch, what am i doing wrong?